l10n_ec_delivery_note/models/delivery_note.py

```diff
@@ -0,0 +1,583 @@
+        size=20,
+        readonly=True,
+        index=True,
+        tracking=True,
+        compute="_compute_document_number",
+        store=True,
+    )
+    journal_id = fields.Many2one(
+        comodel_name="account.journal",
+        string="Emission Point",
+        readonly=True,
+        states=STATES,
+        check_company=True,
+        domain=[("l10n_latam_use_documents", "=", True), ("type", "=", "sale")],
+    )
+    transfer_date = fields.Date(
+        required=True,
+        readonly=True,
+        states=STATES,
+        default=lambda self: fields.Date.context_today(self),
+        tracking=True,
+    )
+    delivery_date = fields.Date(
+        required=True,
+        readonly=True,
+        states=STATES,
+        default=lambda self: fields.Date.context_today(self),
+        tracking=True,
+    )
+    motive = fields.Text(readonly=True, states=STATES, copy=False)
+    l10n_ec_car_plate = fields.Char(
+        "Car plate", size=8, required=False, readonly=True, states=STATES
+    )
+    stock_picking_ids = fields.Many2many(
+        "stock.picking",
+        string="Pickings related",
+        readonly=True,
+        states=STATES,
+        copy=False,
+    )
+    sale_order_ids = fields.Many2many(
+        "sale.order",
+        "sale_order_delivery_note_rel",
+        "delivery_note_id",
+        "order_id",
+        "Sales Order",
+        readonly=True,
+        copy=False,
+    )
+    partner_id = fields.Many2one(
+        "res.partner",
+        "Partner",
+        readonly=True,
+        states=STATES,
+        index=True,
+        auto_join=True,
+        tracking=True,
+    )
+    commercial_partner_id = fields.Many2one(
+        "res.partner",
+        string="Commercial Entity",
+        compute_sudo=True,
+        related="partner_id.commercial_partner_id",
+        store=True,
+        readonly=True,
+        help="The commercial entity that will be used on Journal Entries for this invoice",
+    )
+    delivery_address_id = fields.Many2one(
+        "res.partner",
+        "Delivery Address",
+        readonly=True,
+        states=STATES,
+        index=True,
+        tracking=True,
+    )
+    delivery_carrier_id = fields.Many2one(
+        "res.partner",
+        "Delivery Carrier",
+        readonly=True,
+        states=STATES,
+        index=True,
+        domain=[("l10n_ec_is_carrier", "=", True)],
+    )
+    delivery_note_type = fields.Selection(
+        [
+            ("sales", "Transfer by Sales"),
+            ("internal", "Internal Transfer"),
+        ],
+        readonly=True,
+        states=STATES,
+        default="sales",
+    )
+    delivery_line_ids = fields.One2many(
+        "l10n_ec.delivery.note.line",
+        "delivery_note_id",
+        "Delivery note detail",
+        readonly=True,
+        states=STATES,
+        copy=True,
+        auto_join=True,
+    )
+    state = fields.Selection(
+        [
+            ("draft", "Draft"),
+            ("waiting", "Pending"),
```

Comment:
Waiting state is related with EDI process?

Reply:
done